detailed_docs/UnixSignals/LowVoltage/LowVoltage_UC.md

```diff
@@ -0,0 +1,105 @@
+## Use Case 1: Handling LOW_VOLTAGE event signals
+
+**Main Flow:**
+
+_Pre-conditions:_   
+a.	SDL and HMI are started  
+b.	Mobile application is running on SDL in FULL/LIMITED HMI level  
+c.	Mobile application has some data that can be resumed
+
+_Steps:_
+
+1.	The battery voltage hits below a certain predefined threshold set by the system
+2.	SDL gets LOW_VOLTAGE signal from HMI
+
+_Expected:_   
+3.	SDL stops all read write activities, audio/video streaming, ignores all requests from application and responses/messages from HMI  
+4.	Voltage is restored  
+5.	Next ignition cycle starts  
```

Comment:
5.	Next ignition cycle starts   - redundant, 
WAKE_UP signal should be received during one ign cycle